### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by olegrise**
## Description
Update readme - add description of the provided data.

## Motivation and Context
This information is going to help designers with component usage.

## How Has This Been Tested?
N/A

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +46,48 @@ The component listens for the following events:
...
+- **statistics**:
+    - **retweetCount**: The number of retweets of the current tweet
+    - **likeCount**: The number of likes to the current tweet
+- **user**:
+    - **description**: The description of the account, as indicated by its owner
+    - **statuses**: The number of statuses/tweets of the account
+    - **followers**: The number of followers of the account
+- **quoted**: An object with the same structure as a tweet (name, screenName, etc, but not another quoted item). Only available if the main tweet is a quote to another tweet.
+
+#### Example
```
Comment (stulees) [inline README.md:67]: @olegrise I think we should show example values for `images` and `quoted`. This PR https://github.com/Rise-Vision/twitter-service/pull/23 I did a while back for the service finalized the formatting of data. I provided two examples in the PR that pertain to `images` and `quoted` being populated. Have a look at those for reference (fyi the example links are calling our previous _get-tweets_ endpoint but it's okay).

Comment (olegrise) [reply README.md:67]: @stulees added. Please review the latest commit https://github.com/Rise-Vision/rise-data-twitter/pull/24/commits/f12b670d3eb3ecc15220c91e14cb333323da7ab0. Thanks.

File: README.md [hunk 2]
```diff
@@ -48,7 +46,67 @@ The component listens for the following events:
...
-_To be defined_
+The response includes an array of objects where each object contains the following fields:
+
+- **name**: The title of the account, as indicated by its owner
+- **screenName**: The account name/handle
+- **profilePicture**: The url to the largest available version of the profile picture
+- **text**: The full text of the tweet
+- **createdAt**: The date when the tweet was created
+- **images**: A list of urls with the largest image size available 
+- **media**: A list of urls with media other than images (video and external links)
```
Comment (stulees) [inline README.md:57]: @olegrise We don't provide `media` property, can you remove, thanks

Comment (olegrise) [reply README.md]: @stulees done. Thanks.

**Pull request merged**